opentech/static_src/src/sass/apply/components/_status-block.scss

```diff
@@ -0,0 +1,74 @@
+.status-block {
+    $root: &;
+    background-color: $color--white;
+    border: 1px solid $color--light-mid-grey;
+
+    @include media-query(tablet-landscape) {
+        display: flex;
+    }
+
+    &__item {
+        padding: 20px;
+        border-bottom: 1px solid $color--light-mid-grey;
+        background-color: $color--white;
+        transition: background-color $quick-transition;
+
+        @include media-query(tablet-landscape) {
+            border-bottom: 0;
+            border-right: 1px solid $color--light-mid-grey;
+            width: calc(100% / 9);
+            padding: 10px;
+        }
+
+        @include media-query(desktop) {
+            padding: 20px 20px 30px;
+        }
+
+        @include media-query(laptop-short) {
+            padding: 15px 15px 25px;
+        }
+
+        &:hover {
+            background-color: $color--light-grey;
+        }
+
+        &:last-child {
+            border-right: 0;
+        }
+    }
+
+    &__info,
+    &__title {
+        margin: 0 0 5px;
+    }
+
+    &__info {
+        font-size: 14px;
+        color: $color--dark-blue;
+    }
+
+    &__title {
+        font-weight: $weight--semibold;
+
+        @include media-query(tablet-landscape) {
+            height: 55px;
```

Comment:
Guessing this ensures an even height for titles as if they were on 2 lines. Do you know for sure that all titles will wrap for no more than 2 lines? Could use min-height instead - might be more accessible if text size gets bumped up.